client/blocks/import/list/index.tsx

```diff
@@ -64,7 +70,10 @@ export default function ListStep( props: Props ) {
 	};
 
 	const recordImportList = () => {
-		recordTracksEvent( trackEventName, trackEventParams );
+		if ( ! skipTracking ) {
```

Comment:
This step probably needs to be updated. 
Stepper already triggers the `calypso_signup_step_start` automatically. 
If you check the network tab on developer tools and search for `calypso_signup_step_start` you can see two calls to the same event as one of them is using a hardcode value on the "flow" attribute.  

https://github.com/Automattic/wp-calypso/blob/f3ce6703a080dcfc3636bb756743ba2b3a84db3c/client/blocks/import/list/index.tsx#L17

I just don't removed it because we can have reports using it.